product_docs/docs/postgres_distributed_for_kubernetes/1/architecture.mdx

```diff
@@ -192,3 +192,9 @@ reliably communicate with each other.
 can be realized on multiple Kubernetes clusters that meet the connectivity
 requirements.
 More information can be found in the ["Connectivity"](connectivity.md) section.
+
+!!! NoteAvailability zones 
+    When creating Kubernetes clusters in different availability zones for cross-regional replication, 
+    ensure the clusters can communicate with each other by enabling network connectivity. To ensure 
+    pods and services are reachable by all Kubernetes clusters, deploy a network connectivity application like 
+    [Submariner](https://submariner.io/) on every cluster. 
```

Comment:
@jlong49 on one of your demos, you explained that apart from setting up submariner, you need to apply a `ServiceExport` YAML object to ensure data is replicated between clusters. 

Is that something we need to add here? Or maybe even as an additional step in this quickstart guide?

Reply:
the serviceexport is part of submariner we can explain the more general purpose of what the serviceexport does in case they use some other network connectivity solution.